libraries/standard/mqtt/src/iot_mqtt_api.c

```diff
@@ -117,18 +117,20 @@ static bool _createKeepAliveOperation( const IotMqttNetworkInfo_t * pNetworkInfo
  * @param[in] pNetworkInfo User-provided network information for the connection
  * connection.
  * @param[out] pNetworkConnection On success, the created and/or initialized network connection.
+ * @param[out] pCreatedNetworkConnection On success, true if a new network connection was created; false if an existing one will be used
```

Comment:
Nit: period at end of comment and backticks around true and false for consistency. Are we sure we want to keep the `pCreatedNetworkConnection` name?

Reply:
Fixed backticks & period.  Changed parameter name to `pCreatedNewNetworkConnection`.